WISE-Community/WISE-Client
# feat(OpenResponse): Add FeedbackRule authoring

Part 2 of work for issue #829 "feat(OpenResponse): Add FeedbackRuleEvaluator". These are planned future PR's for this issue:
- refactoring, organizing folders to share common classes between OR and DG

## Changes
- Add FeedbackRule authoring to OpenResponse advanced component editing view
- Remove ScoringRule authoring from OpenResponse advanced component editing view
- Remove old translation strings

## Test
In the OpenResponse advanced component editing view:
- Verify that "Enable Feedback Rules" checkbox appears iff CRater is enabled
- Verify that "Scoring Rule" authoring section no longer appears when CRater is enabled
- Check "Enable Feedback Rules" checkbox
   - Verify that "isDefault" feedback is displayed as initial rule
   - Verify that the help button launches the Feedback Rule Authoring help dialog
   - Test that you can add, delete, reorder feedback rules as you can in DialogGuidance
      - Multiple feedback per rule is allowed (you can use the "+" to add more feedback per rule) 
- Un-check "Enable Feedback Rules" checkbox
   - Verify that FeedbackRules is not used 
   - If you enable the checkbox again, the rules that you authored before should appear again

In the OpenResponse student view:
- Test that the Feedback Rules that you authored above work as expected
